.bundle/config

```diff
@@ -1,3 +1,5 @@
 ---
 BUNDLE_PATH: "vendor/bundle"
+BUNDLE_RETRY: "3"
+BUNDLE_JOBS: "3"
```

Comment:
are these related to this specific PR?

Reply:
No, I missed that this got committed. I'm not sure why it changed exactly (guessing a different bundler version) so I'll revert it.